tester/orb.yaml

```diff
@@ -188,7 +219,14 @@ jobs:
       - run:
           working_directory: <<parameters.cwd>>
           command: docker-compose <<parameters.compose_args>> run --rm <<parameters.service_name>> <<parameters.args>>
+          environment:
+            # Workaround for docker-compose being unable to pull image from gcr
```

Comment:
Note: if you do pull the GCR stuff out, this is OK to keep -- you've set this variable to its standard value, so that should be completely reasonable in all cases (I'm surprised `cimg` doesn't already have this set...)